pkgs/tools/inputmethods/fcitx/fcitx-qt5.nix

```diff
@@ -32,11 +23,10 @@ stdenv.mkDerivation rec {
   '';
 
   meta = with stdenv.lib; {
-    homepage    = https://github.com/fcitx/fcitx-qt5;
+    homepage    = "https://gitlab.com/fcitx/fcitx-qt5";
```

Comment:
Not needed

Reply:
the quotes I mean

Reply:
Thanks for the review!
Updated as your suggestion.